.prettierignore

```diff
@@ -0,0 +1,6 @@
+/ui/public
+/db/
+/conf/
+
+# TODO re-write from scratch or fix all html structure issues
+/lib/notification/emailTemplate/template.hbs
```

Comment:
Neither me nor any parser/GPT could figure out how to restructure it, hence the ignore and TODO here 😂